### PR TITLE
feat: COMMENT ON graph objects

### PR DESCRIPTION
**Pull request by gtyun**
Enable COMMNET ON GRAPH/VLABEL/ELABEL
Add default comment for base graph labels and tables.
Descriptions will be shown with
* \dG+ graph
* \dGL+ all kind of label
* \dGE+ edge label
* \dGV+ vertex label

### CODE REVIEW

File: src/backend/catalog/ag_label.c [hunk 1]
```diff
@@ -99,3 +101,48 @@ InsertAgLabelTuple(Relation ag_label_desc, Oid labid, RangeVar *label,
 
 	heap_freetuple(tup);
 }
+
```
Comment (protodef) [inline src/backend/catalog/ag_label.c]: Is this the right place for these functions? I think these functions should go to `lsyscache.c`.

File: src/backend/catalog/ag_label.c [hunk 2]
```diff
@@ -99,3 +101,48 @@ InsertAgLabelTuple(Relation ag_label_desc, Oid labid, RangeVar *label,
...
+	Form_ag_label labtup;
+
+	tuple = SearchSysCache1(LABELOID, ObjectIdGetDatum(labid));
+
+	if (!HeapTupleIsValid(tuple))
+		elog(ERROR, "cache lookup failed for label (OID=%u)", labid);
+
+	labtup = (Form_ag_label) GETSTRUCT(tuple);
+
+	result = NameStr(labtup->labname);
```
Comment (protodef) [inline src/backend/catalog/ag_label.c]: You should return a copy of the `labname`.

File: src/backend/commands/graphcmds.c [hunk 3]
```diff
@@ -261,32 +261,24 @@ AgInheritanceDependancy(Oid labid, List *supers)
 void
 CheckDropLabel(ObjectType removeType, Oid labid)
 {
-	HeapTuple	tuple;
-	Form_ag_label labtup;
+	char	 labkind = get_labkind_with_labid(labid);
```
Comment (protodef) [inline src/backend/commands/graphcmds.c]: These two functions are called only at here. Is it necessary to use these functions to look up the same cache entry twice?

Comment (gtyun) [inline src/backend/commands/graphcmds.c]: I made a new error check logic but removed it. One of this was called by that logic. I will restore CheckDropLabel.

**Comment by protodef** [comment]
I've reviewed your commit, fixed some bugs, and clean up codes. Please review the commit I pushed. If it looks fine, I will merge it to the master.

**Comment by protodef** [comment]
I've rebased and merged it.

**Pull request closed without merging**